dist/vue-currency-filter.cjs.js

```diff
@@ -0,0 +1,359 @@
+'use strict';
```

Comment:
Can you delete all files under `dist/` folder.
It should be in our `.gitignore`